docs/modules/ROOT/partials/nav.adoc

```diff
@@ -39,6 +40,7 @@
 *** xref:oc4:ROOT:how-tos/cloudscale/decommission.adoc[Decommissioning]
 
 ** Exoscale
+*** xref:oc4:ROOT:references/exoscale/architecture.adoc[Architecture]
```

Comment:
```suggestion
```

Or do the same for vSphere for consistency. I prefer to not have the same page at multiple locations in the navigation. It is confusing.

Reply:
The current state is that we have links to many (most?) pages in kb.vshn.ch/oc4 in multiple places in the navigation, since we restructured the navigation. Also, given the current navigation structure, there's no real way to avoid this (as far as I can tell/remember from the last time I looked at it)